src/seismometer/table/analytics_table.py

```diff
@@ -0,0 +1,715 @@
+import itertools
```

Comment:
low priority; While you are here, is it reasonable to move fairness.py to the tables folder, since they both make tables?

Reply:
Moved it to the table subpackage.